charts/kubearchive/templates/api_server/api_server.yaml

```diff
@@ -48,7 +48,10 @@ spec:
               value: {{ tpl .Values.database.url . }}
             - name: POSTGRES_PORT
               value: "{{ .Values.database.service.port }}"
-{{ include "kubearchive.v1.otel.env" .Values.apiServer | indent 12 }}
+            - name: KUBEARCHIVE_OTEL_ENABLED
+              value: '{{ ternary "false" "true" (eq .Values.observability.endpoint "") }}'
+            - name: OTEL_EXPORTER_OTLP_ENDPOINT
+              value: "{{ tpl .Values.observability.endpoint . }}"
```

Comment:
Since the same think added here is also being added to the sink, i think that it makes sense to have a template for this (like was defined in `_helpers.tpl`

Reply:
Added the include back.